breathecode/monitoring/actions.py

```diff
@@ -361,3 +361,54 @@ def download_csv(module, model_name, ids_to_download, academy_id=None):
         download.status_message = str(e)
         download.save()
         return False
+
+
+def upload_csv(module, model_name, file_content, academy_id=None):
```

Comment:
Check if it is used, check if fix it or deleted if it is unused

Reply:
In the issue https://github.com/4GeeksAcademy/About-4Geeks-Academy/issues/2576 dice que lo necesitamos?

Reply:
I think under this conditions we can delete this function